system-int-test/test/cmr/system_int_test/search/facets/collection_facets_v2_search_test.clj

```diff
@@ -520,6 +520,14 @@
     (testing value
      (is (= count (:count field-match-value))))))
 
+(defn- assert-latency-facet-field
+   "Assert the latency facet field with count and link match the facets result"
+   [facets-result value count link]
```

Comment:
We shouldn't shadow the `count` function with a variable name, maybe see about renaming that variable

Reply:
We have several functions that take the count as a parameter. I don't really see how it could impact that count function.

Reply:
It's not that it stops the count function from being used but it does go contrary to the style guide
https://guide.clojure.style/#dont-shadow-clojure-core